demisto_sdk/commands/common/hook_validations/image.py

```diff
@@ -24,19 +23,24 @@ def __init__(self, file_path, ignored_errors=None, print_as_warnings=False):
         super().__init__(ignored_errors=ignored_errors, print_as_warnings=print_as_warnings)
         self._is_valid = True
         self.file_path = ''
-
-        if checked_type(file_path, INTEGRATION_REGXES) or re.match(IMAGE_REGEX, file_path, re.IGNORECASE):
+        if file_path.endswith('.png'):
             self.file_path = file_path
+        # For integrations that are not in a package format, the image is within the yml
         else:
-            if checked_type(file_path, YML_INTEGRATION_REGEXES):
+            data_dictionary = get_yaml(file_path)
+            if not data_dictionary:
+                return
+            # For old integration in which image is inside the yml.
+            elif data_dictionary.get('image', ''):
+                self.file_path = file_path
+            # For new integrations -  Get the image from the folder.
+            else:
                 try:
                     self.file_path = glob.glob(os.path.join(os.path.dirname(file_path), '*.png'))[0]
                 except IndexError:
                     error_message, error_code = Errors.no_image_given()
                     if self.handle_error(error_message, error_code, file_path=self.file_path):
                         self._is_valid = False
```

Comment:
half related. I do not think we cover this exception part in the ut. can you check yana, and if not can you add a test case of an pack without a given .png file?

Reply:
added